### PR TITLE
Feature/queue command

### PR DESCRIPTION
**Pull request by sagikazarmark**
Adds the command wrapper mentioned in #21

### CODE REVIEW

**Comment by sagikazarmark** [comment]
**Note:** it depends on #27

File: src/QueueCommand.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace League\Tactician\Bernard;
+
+/**
+ * Wraps any command to be queueable
+ *
+ * @final
```
Comment (rosstuck) [inline src/QueueCommand.php:8]: Might as well just make it actually final, eh? :smile_cat:

**Comment by rosstuck** [comment]
Looks alright to me. :+1:

File: src/QueueCommand.php (hunk 1 above)
Comment (sagikazarmark) [inline src/QueueCommand.php:8]: Yeah ;)

**Pull request merged**